raiden/blockchain/abi.py

```diff
@@ -51,7 +51,7 @@
 CONTRACT_NETTING_CHANNEL = 'NettingChannelContract'
 CONTRACT_REGISTRY = 'Registry'
 
-CONTRACT_TOKEN_NETWORK_REGISTRY = 'TokenNetworkRegistry'
+CONTRACT_TOKEN_NETWORK_REGISTRY = 'TokenNetworksRegistry'
```

Comment:
The name should be fixed in the smart contracts, not here, as suggested by Lefteris. Issue: https://github.com/raiden-network/raiden-contracts/issues/107

Reply:
yep, please switch to singular